go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/golang/protobuf v1.4.2
 	github.com/google/uuid v1.1.1
 	github.com/hashicorp/go-multierror v1.0.0 // indirect
-	github.com/kubernetes-csi/csi-proxy/client v0.2.1
+	github.com/kubernetes-csi/csi-proxy/client v0.2.2
```

Comment:
Can you update the README as well to indicate we have a newer csi proxy dependency?

Reply:
yes, I will have another PR to update it.